brave/brave-core
# Adds screenshot options to context menu and commands

Resolves https://github.com/brave/brave-browser/issues/25816
Security/Privacy Review: https://github.com/brave/reviews/issues/1831

https://github.com/user-attachments/assets/f39707f4-a129-41ba-ae2c-c79972ebad92



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Right-click on a page
2. Click _Screenshot Tools_ from the context menu
3. Select an option (i.e., Selection, Viewport, Full page) from the submenu

Once a screenshot has been taken, you should see a UI element (with a preview of your screenshot) informing you that the screenshot has been copied to your device's clipboard.
